ch_backup/cli.py

```diff
@@ -100,6 +122,17 @@ def signal_handler(signum, _frame):
     help="Disable certificate verification for https protocol.",
 )
 @option("--zk-hosts", type=str, help="Use specified zk hosts for connection to the ZK")
+@option(
+    "--config-value",
+    "config_values",
+    multiple=True,
+    type=(str, JsonParamType()),
+    metavar="NAME VALUE",
+    help="Paths and values to override ch-backup config values. "
+    'Path should be a string separated with any of following symbols ".,:". '
```

Comment:
I think we can support only `.` as a separator for simplicity.

In dbaas tool several options are supported for compatibility with salt stack and postgresql syntaxes. Not sure that this feature is actual for ch-backup.

Reply:
Ok, dot is enough, because I am mentioning it in "help" explicitly.
But actually it does not simplify code much :)